packages/web-component/src/app/index.html

```diff
@@ -28,9 +28,18 @@
         flow-id="<flow-id>"
         base-url="<base-url>"
         debug="true"
+        error-transformer="translateError"
```

Comment:
We should use a property instead of attribute in this case
https://open-wc.org/guides/knowledge/attributes-and-properties/